packages/app/components/BetaNFTModal.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  BodyText,
+  Button,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  HeadingText,
+  ButtonLink,
+  ModalFooter,
+} from "@/ui";
+import { ModalId, useModalContext } from "@/contexts";
+import { useDisconnect } from "wagmi";
+import { DiscordIcon, NFT } from "@/public/assets";
```

Comment:
Are we missing where to display this `DiscordIcon`?
I see the logic in the `ButtonLink` but dunno if we need the icon for another purpose here

Reply:
true, that is not needed